source/community/index.html

```diff
@@ -6,8 +6,6 @@
 menu_title: Community
 menu_order: 35
 menu_style: bullet
-redirect:
```

Comment:
why did you removed the redirects? this will cause breaks for old links, I guess

Reply:
The sculpin redirect link bundle is broken with newer versions and since this is not maintained by someone which we knew and these are just a few links, I decided to remove these few.